main.scss

```diff
@@ -7,4 +9,23 @@
 
 	// Set to silent again to avoid being output twice
 	$o-comments-is-silent: true !global;
+
+	.o-comments__displayname-prompt {
+		width: 100%;
+		max-width: 400px;
+
+		.o-comments__displayname-container {
+			margin-top: 8px;
```

Comment:
It would be great to use `o-spacing`. Encourages more consistent spacing and allows us to toggle to relative units.

Reply:
Opps, yes I forgot that was a thing